external-dependencies.gradle

```diff
@@ -7,7 +7,8 @@ ext {
             "io.dropwizard:dropwizard-auth:${dropwizard_version}",
             "io.dropwizard:dropwizard-configuration:${dropwizard_version}",
             "io.dropwizard:dropwizard-jdbi:${dropwizard_version}",
-            "io.dropwizard:dropwizard-logging:${dropwizard_version}"
+            "io.dropwizard:dropwizard-logging:${dropwizard_version}",
+            "io.dropwizard:dropwizard-client:${dropwizard_version}"
```

Comment:
nitpick: can you maintain the alphabetical order? it makes for easier scanning through lists of deps later